test/integration_test.py

```diff
@@ -607,7 +606,18 @@ def test_azul_client_error_handling(self):
                           notifications)
 
 
-class PortalRegistrationIntegrationTest(IntegrationTestCase):
+class PortalRegistrationIntegrationTest(IntegrationTestCase, AlwaysTearDownTestCase):
+    # Currently takes about 50 seconds and creates a 25 kb db file.
```

Comment:
why did you convert these to attributes?

Reply:
I don't remember my reasoning and no longer see a need for this change.